Build fails on nightly (2018-04-26)

I confirmed that it builds on `nightly-2018-04-19`, which seems like it was the last nightly release. Here are the errors:
```
error[E0658]: procedural macros cannot be expanded to expressions
  --> maud_macros/src/build.rs:28:17
   |
28 |                 quote!($output_ident.push_str($string);)
   |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: add #![feature(proc_macro_non_items)] to the crate attributes to enable

error[E0658]: procedural macros cannot be expanded to expressions
  --> maud_macros/src/build.rs:60:19
   |
60 |           self.push(quote!({
   |  ___________________^
61 | |             // Create a local trait alias so that autoref works
62 | |             trait Render: maud::Render {
63 | |                 fn __maud_render_to(&self, output_ident: &mut String) {
...  |
68 | |             $expr.__maud_render_to(&mut $output_ident);
69 | |         }));
   | |__________^
   |
   = help: add #![feature(proc_macro_non_items)] to the crate attributes to enable

error[E0658]: procedural macros cannot be expanded to expressions
   --> maud_macros/src/build.rs:123:19
    |
123 |         self.push(quote!(if $cond { $body }));
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = help: add #![feature(proc_macro_non_items)] to the crate attributes to enable

error[E0658]: procedural macros cannot be expanded to expressions
  --> maud_macros/src/lib.rs:43:5
   |
43 | /     quote!({
44 | |         extern crate maud;
45 | |         let mut $output_ident = String::with_capacity($size_hint);
46 | |         $stmts
47 | |         maud::PreEscaped($output_ident)
48 | |     })
   | |______^
   |
   = help: add #![feature(proc_macro_non_items)] to the crate attributes to enable
```

--- a/maud/tests/basic_syntax.rs
+++ b/maud/tests/basic_syntax.rs
@@ -1,5 +1,6 @@
 #![feature(plugin)]
 #![feature(proc_macro)]
+#![feature(proc_macro_non_items)]
 
 #![plugin(maud_lints)]
 

--- a/maud/tests/control_structures.rs
+++ b/maud/tests/control_structures.rs
@@ -1,5 +1,6 @@
 #![feature(plugin)]
 #![feature(proc_macro)]
+#![feature(proc_macro_non_items)]
 
 #![plugin(maud_lints)]
 

--- a/maud/tests/misc.rs
+++ b/maud/tests/misc.rs
@@ -1,5 +1,6 @@
 #![feature(plugin)]
 #![feature(proc_macro)]
+#![feature(proc_macro_non_items)]
 
 #![plugin(maud_lints)]
 

--- a/maud/tests/splices.rs
+++ b/maud/tests/splices.rs
@@ -1,5 +1,6 @@
 #![feature(plugin)]
 #![feature(proc_macro)]
+#![feature(proc_macro_non_items)]
 
 #![plugin(maud_lints)]
 

--- a/maud_extras/lib.rs
+++ b/maud_extras/lib.rs
@@ -1,5 +1,6 @@
 #![feature(plugin)]
 #![feature(proc_macro)]
+#![feature(proc_macro_non_items)]
 
 #![plugin(maud_lints)]
 
@@ -13,6 +14,7 @@ use maud::{Markup, Render, html};
 ///
 /// ```rust
 /// # #![feature(proc_macro)]
+/// # #![feature(proc_macro_non_items)]
 /// # extern crate maud;
 /// # extern crate maud_extras;
 /// # use maud::html;
@@ -39,6 +41,8 @@ impl<T: AsRef<str>> Render for Css<T> {
 ///
 /// ```rust
 /// # #![feature(proc_macro)]
+/// # #![feature(proc_macro_non_items)]
+/// #
 /// # extern crate maud;
 /// # extern crate maud_extras;
 /// # use maud::html;
@@ -65,6 +69,7 @@ impl<T: AsRef<str>> Render for Js<T> {
 ///
 /// ```rust
 /// # #![feature(proc_macro)]
+/// # #![feature(proc_macro_non_items)]
 /// # extern crate maud;
 /// # extern crate maud_extras;
 /// # use maud::html;
@@ -91,6 +96,7 @@ impl<T: AsRef<str>, U: AsRef<str>> Render for Meta<T, U> {
 ///
 /// ```rust
 /// # #![feature(proc_macro)]
+/// # #![feature(proc_macro_non_items)]
 /// # extern crate maud;
 /// # extern crate maud_extras;
 /// # use maud::html;
@@ -117,6 +123,7 @@ impl<T: AsRef<str>> Render for Title<T> {
 ///
 /// ```rust
 /// # #![feature(proc_macro)]
+/// # #![feature(proc_macro_non_items)]
 /// # extern crate maud;
 /// # extern crate maud_extras;
 /// # use maud::html;
@@ -143,6 +150,7 @@ impl<T: AsRef<str>> Render for Charset<T> {
 ///
 /// ```rust
 /// # #![feature(proc_macro)]
+/// # #![feature(proc_macro_non_items)]
 /// # extern crate maud;
 /// # extern crate maud_extras;
 /// # use maud::html;
@@ -169,6 +177,7 @@ impl<T: AsRef<str>, U: AsRef<str>> Render for MetaProperty<T, U> {
 ///
 /// ```rust
 /// # #![feature(proc_macro)]
+/// # #![feature(proc_macro_non_items)]
 /// # extern crate maud;
 /// # extern crate maud_extras;
 /// # use maud::html;

--- a/maud_macros/src/lib.rs
+++ b/maud_macros/src/lib.rs
@@ -1,4 +1,5 @@
 #![feature(proc_macro)]
+#![feature(proc_macro_non_items)]
 
 #![doc(html_root_url = "https://docs.rs/maud_macros/0.17.3")]
 

